server/util/disk/disk.go

```diff
@@ -76,12 +83,19 @@ func FileExists(fullPath string) (bool, error) {
 	}
 }
 
-type SectionReaderCloser struct {
+type readCloser struct {
 	*io.SectionReader
 	io.Closer
+	ctx context.Context
 }
 
-func FileReader(ctx context.Context, fullPath string, offset, length int64) (*SectionReaderCloser, error) {
+func (r *readCloser) Read(p []byte) (int, error) {
+	_, spn := tracing.StartSpan(r.ctx)
```

Comment:
ctx, spn := tracing.StartSpan(r.ctx)

Reply:
So when I went to do this, I realized there is no context variable referenced in this function, which means I can't name the variable. Same for below.

Reply:
lol i'm a goof, thanks for looking :)